README.md

```diff
@@ -35,11 +35,13 @@ iOS security related tools
 
 A collection of ios security related resources
 
-1. [IDB](https://github.com/dmayer/idb) - iOS App Security Assessment Tool
-2. [iRET](https://github.com/S3Jensen/iRET) - iOS Reverse Engineering Toolkit
-3. [DVIA](http://damnvulnerableiosapp.com/) - Damn Vulnerable iOS App for learning
-4. [LibiMobileDevice](https://github.com/libimobiledevice/libimobiledevice) - A cross-platform protocol library to communicate with iOS devices
-5. [Needle](https://github.com/mwrlabs/needle) - iOS App Pentesting Tool
+1. [AppCritique](https://appcritique.boozallen.com) - Upload your iOS application and receive a FREE comprehensive security assessment.
```

Comment:
1. Why not add AppCritique at the bottom of the list?
2. "Upload your iOS application and receive a FREE comprehensive security assessment." this looks like a marketing speak, can you change this to "iOS App Security Assessment Tool" instead?

Reply:
I thought I read somewhere that it needed to be in alphabetic order. I will make the changes! Thanks